.travis.yml

```diff
@@ -1,12 +1,12 @@
 language: python
 python:
-  - "2.7"
+  - "3.7"
```

Comment:
Are you specifically breaking 2.x support?

Reply:
Yes, I’m excluding python 2 in the list of supported versions under setup.py and creating a new major version of kingpin to signify there is a backwards incompatible change.